versions.gradle

```diff
@@ -3,7 +3,7 @@
 ext {
     versionCode = 43
     versionName = '1.11.3'
-    minSdkVersion = 27
+    minSdkVersion = 17
```

Comment:
Our SDK is min API level 16, where does that 17 comes from?

Reply:
Good catch. reverted to wrong version. Fixed to 16.